### PR TITLE
update firefox-beta to 50.0b5;  add more languages.

### PR DESCRIPTION
**Pull request by yurikoles**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by yurikoles** [comment]
How to verify download of specific language of the cask?

**Comment by reitermarkus** [comment]
> How to verify download of specific language of the cask?

That's not possible at the moment. For now you'll either have to comment out all other languages or re-download everything.

https://github.com/Homebrew/brew/pull/1264

**Comment by reitermarkus** [comment]
Actually, `brew cask fetch` should also verify the download, for example you can run:

```
brew cask fetch firefox-beta --language=it
```

**Comment by fanquake** [comment]
@yurikoles Is there anything left todo here?

File: Casks/firefox-beta.rb [hunk 1]
```diff
@@ -1,42 +1,77 @@
...
   end
 
+  language 'zh-TW' do
+    sha256 '31bf770c94b5c4ac4df6c1da96c821ca0bafd4edaada9900e32abc86179e9efb'
+    'zh-TW'
+  end
+
   language 'zh' do
-    sha256 'f61a688e4ac032f89e2d4f382f7beb3c2489dfea3051d8b741fc474d6188dd36'
+    sha256 'd654f8be4e0de57f2902f6d56988ddd9a548b23ebb9eea9a46b4fa5b3b3c5732'
```
Comment (jawshooah) [inline Casks/firefox-beta.rb]: Looks like this is now out of date:

```
==> Auditing language: 'zh'
==> Downloading https://download.mozilla.org/?product=firefox-50.0b5-SSL&os=osx&lang=zh-CN
######################################################################## 100.0%
==> Verifying checksum for Cask firefox-beta
==> Note: running "brew update" may fix sha256 checksum errors
audit for firefox-beta: failed
 - download not possible: sha256 mismatch
Expected: d654f8be4e0de57f2902f6d56988ddd9a548b23ebb9eea9a46b4fa5b3b3c5732
Actual: aecdbdc48e970cf3c4c451fe6fd569781f1179566534014d0735196ecf3b8890
```

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 683083ae89f0ead219d5e841fc50a83780da632a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**